### ISSUE TITLE
CR "Stop Onboarding" functionality should be implemented against "View New Hire" screens

### ISSUE THREAD

**Issue by MustafaASK** [ShaikASK/Testing#355]
Here below are the detailed requirement about 'Stop Onboarding' functionality 👍 

- There should be an option for HR user to ‘Stop Onboarding’ of a New Hire during the Onboarding process. 
- This ‘Stop Onboarding’ option is required – In case Candidate is not interested  to join the organization or not responding to the Onboarding process for any reason.
- This ‘Stop Onboarding’ option should be enabled against a New Hire ONLY after initiating the Onboarding of that New Hire.
- HR user Should be able to ‘Stop Onboarding’ even after the New Hire status is ‘completed’.
ONLY HR Admin users should be able to assign this ‘Stop Onboarding’ permission for the selected HR user from ‘Add/Edit HR user’ screen.
- HR users should able to see this ‘Stop Onboarding’ option in the ‘View New Hire’ screen against the selected Candidate.  
- When HR user selects ‘Stop Onboarding’ option from View New Hire screen against the New Hire -> 
	- There should be a pop screen where HR MUST select the ‘Reason for Rejection’ 
	Able to enter the detailed ‘Comments’ for selecting this Reason and stopping the Onboarding 
        process
	- Provision to upload minimum three support documents with a max of 10 MB size in total. There is 
         no restriction in the documents type uploaded.
- After selecting the ‘Submit’ action from Stop Onboarding screen - There should be an email sent to the - Candidate, and BCC’ing HR users and HR Admins.

Mon 17-Dec-18 5:42 PM
The mail will be sent to HR Admins and HR Users only.
The character limit for Comments For Rejection (editor) – 2000.
The Stop Onboarding reason should be shown as tooltip on NewHire List thumbnails.
A signature will be added dynamically for the mails.
Stop Onboard New Hire cannot login the portal.
Maximum of 3 attachments are allowed without any restriction of attached file type.
NewHire Details attachment besides the Stop Onboarding attachments will be sent in the mail.

**Comment by MustafaASK** [comment]
Mahesh Varala - We are marking the defect as "Deferred" as this will be targeted to future release

**Comment by MustafaASK** [comment]
Mahesh Varala - This functionality is now implemented , Hence marking this to close
